doc/graph/fusion_patterns/binary_fusion_patterns.md

```diff
@@ -0,0 +1,45 @@
+Binary Fusion Patterns {#dev_guide_graph_binary_fusion_patterns}
+===========================================================
+
+## Overview
+
+The Binary category includes operations such as: Add, Subtract, Maximum,
+Minimum, Multiply, Divide.
+
+oneDNN supports various binary fusion patterns to optimize performance and
+reduce memory bandwidth requirements. This document describes the supported
+fusion patterns for Binary operations.
+
+## Pattern Structure
+
+oneDNN defines floating-point Binary fusion patterns as follows.
+The blue nodes are required when defining a Binary fusion pattern while the
+brown nodes are optional.
+
+![Binary pattern](images/binary_pattern.png)
+
+1. **Binary Operation**: Performs the corresponding binary operation between the
+   `src_0` and `src_1` tensors. Refer to the Note in
+   [Fusion Patterns](graph_fusion_patterns.html).
+2. **Epilogue Subgraph**: Optional and can include the following operations:
+   - **Binary Operations**.
+   - **Unary Operations**: Refer to the Note in
+   [Fusion Patterns](graph_fusion_patterns.html).
+
+   Combination Rules:
+
+   ![epilogue subgraph](images/epilogue_subgraph_general_1.png)
+
+   - 0 to 4 Binary or Unary operations are supported in the epilogue subgraph.
+
+## Data Types
+
+oneDNN supports the following combinations of data types for src, weights, bias
+and output:
+
+| src_0        | src_1         | output       |
```

Comment:
Not sure if i missed any thing, but I think we agreed to use src/dst as oneDNN convention.
```suggestion
| src_0        | src_1         | dst       |
```

Reply:
I'm trying to align the name here with the pattern image. Do you think I should change the name on the pattern image?